sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcSchemaIOProviderTest.java

```diff
@@ -131,6 +131,31 @@ public void testPartitionedReadWithExplicitSchema() {
     pipeline.run();
   }
 
+  @Test
+  public void testPartitionedReadWithExplicitLongBounds() {
+    JdbcSchemaIOProvider provider = new JdbcSchemaIOProvider();
+
+    Row config =
+        Row.withSchema(provider.configurationSchema())
+            .withFieldValue("driverClassName", DATA_SOURCE_CONFIGURATION.getDriverClassName().get())
+            .withFieldValue("jdbcUrl", DATA_SOURCE_CONFIGURATION.getUrl().get())
+            .withFieldValue("username", "")
+            .withFieldValue("password", "")
+            .withFieldValue("partitionColumn", "id")
+            .withFieldValue("partitions", (short) 5)
+            .withFieldValue("longLowerBound", 0L)
```

Comment:
nit: leave the type out of the field name

These names are user-facing (unless we do work to hide them) in managed IO, YAML, etc